build.gradle

```diff
@@ -32,6 +32,10 @@ buildscript {
         // Needed for caching reflected data during builds
         classpath 'org.reflections:reflections:0.9.10'
         classpath 'dom4j:dom4j:1.6.1'
+
+        // plugin for compilation Protobuf files
+        // The latest version is 0.8.1, but it requires Gradle 2.12
```

Comment:
I guess we can (should) update our Gradle wrapper so that we can use latest version.